docs/pages/tutorials/minimal.mdx

```diff
@@ -0,0 +1,31 @@
+# Modifying the getting started example
+
+These tutorials teach you how to make various changes to [the getting started example](https://github.com/latticexyz/mud/tree/main/examples/minimal).
+This is an easy way to learn how to modify mud functionality in various ways.
+
+## Getting the initial version
+
+All of these examples are based on the vanilla version of getting started.
+To create the basic version to modify, follow these steps:
+
+1. Create a version 2.0 mud starter.
+
+   ```bash
+   pnpm create mud@canary tutorial
+   ```
+
+1. Select the **vanilla** template.
```

Comment:
somehow all of these are `1.`

Reply:
That is not a problem, markdown numbers them automatically for us. I prefer to have all `1.`s  because that way if I add or remove items I don't need to manually renumber.

Reply:
oh wait really? didn't know about this, that's pretty cool